server/api/controllers/forms/post-create-submission.js

```diff
@@ -153,6 +154,71 @@ module.exports = {
         });
       }
 
+      // Send alert email to leaders if setting is turned on
+      if (formRecord[0].alertType !== 'None') {
+        let emailRecipients;
+
+        // get email addresses of the recipients
+        if (formRecord[0].alertType === 'Custom') {
+          emailRecipients = formRecord[0].customAlertRecipients;
+        } else if (
+          ['LIFE Group', 'Lifestage', 'Campus'].includes(
+            formRecord[0].alertType
+          )
+        ) {
+          const recipientsDict =
+            await sails.helpers.forms.getFormAlertRecipients(formId);
+
+          // get the current season leadership teams
+          const today = new Date();
+
+          let fieldSelected;
+
+          switch (formRecord[0].alertType) {
+            case 'Campus':
+              fieldSelected = 'campus';
+              break;
+            case 'Lifestage':
+              fieldSelected = 'lifestage';
+              break;
+            case 'LIFE Group':
+              fieldSelected = 'lifeGroup';
+              break;
+          }
+
+          const recipientsEntries =
+            recipientsDict[submissionData[fieldSelected]];
+
+          for (const entry of recipientsEntries) {
+            if (
+              today >= new Date(entry.seasonFrom) &&
+              today <= new Date(entry.seasonTo)
+            ) {
+              emailRecipients = entry.leaderEmails;
+            }
+          }
+        }
+
+        // failsafe
+        if (emailRecipients.length <= 0) {
+          emailRecipients = ['web@hongkong.hmcc.net'];
```

Comment:
do we want to do web@hongkong.hmcc.net? then everyone in t3ch would have gotten it ya haahahah

Reply:
where do we want it to go then ya? because ET getting the failed email doesn't make sense too, because they have to tell us which one failed anyway. or just no need to set a failsafe?

Reply:
should we dump it to no-reply